games-action/supertuxkart/supertuxkart-1.2.recipe

```diff
@@ -0,0 +1,102 @@
+an enjoyable experience for all ages.
+
+Explore various coastlines, shipyards, mysterious mansions, and visit the \
+pyramids. Race under water or in space, watching the stars passing \
+by. Have some rest under the palms on the beach (watching the other \
+karts overtaking you :) ). But don't fall in a volcano...
+
+You can do a single race against other karts, compete in one of \
+several Grand Prix, or try to beat the high score in time trials \
+on your own. Play battle mode against the computer, and more. \
+For a greater challenge, race online against players from all over the world \
+and prove your racing skills!"
+HOMEPAGE="https://supertuxkart.net"
+COPYRIGHT="2020 SuperTuxKart Team"
+LICENSE="GNU GPL v3"
+REVISION="1"
+SOURCE_URI="https://github.com/supertuxkart/stk-code/releases/download/$portVersion/supertuxkart-$portVersion-src.tar.xz"
+CHECKSUM_SHA256="052edf0afdbeb99583fe8676fb0ab80ecb6103fb88b7540f858d1b5fa1297d37"
+SOURCE_FILENAME="supertuxkart-$portVersion-src.tar.xz"
+SOURCE_DIR="SuperTuxKart-$portVersion-src"
+
+ARCHITECTURES="!x86_gcc2 x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+ supertuxkart$secondaryArchSuffix = $portVersion
+ app:SuperTuxKart$secondaryArchSuffix
+ "
+
+REQUIRES="
+ haiku$secondaryArchSuffix
+ lib:libcurl$secondaryArchSuffix
+ lib:libenet$secondaryArchSuffix
+ lib:libfreetype$secondaryArchSuffix
+ lib:libGL$secondaryArchSuffix
+ lib:libglu$secondaryArchSuffix
+ lib:libglew$secondaryArchSuffix
+ lib:libharfbuzz$secondaryArchSuffix
+ lib:libjpeg$secondaryArchSuffix
+ lib:libopenal$secondaryArchSuffix
+ lib:libogg$secondaryArchSuffix
+ lib:libpng$secondaryArchSuffix
+ lib:libsdl2_2.0$secondaryArchSuffix
+ lib:libsqlite3$secondaryArchSuffix
+ lib:libssl$secondaryArchSuffix
+ lib:libvorbis$secondaryArchSuffix
+ lib:libz$secondaryArchSuffix
+ "
+
+BUILD_REQUIRES="
+ haiku${secondaryArchSuffix}_devel
+ devel:libcurl$secondaryArchSuffix
+ devel:libenet$secondaryArchSuffix
+ devel:libfreetype$secondaryArchSuffix
+ devel:libGL$secondaryArchSuffix
+ devel:libglu$secondaryArchSuffix
+ devel:libglew$secondaryArchSuffix >= 2
+ devel:libharfbuzz$secondaryArchSuffix
+ devel:libjpeg$secondaryArchSuffix
+ devel:libpng$secondaryArchSuffix
+ devel:libopenal$secondaryArchSuffix
+ devel:libogg$secondaryArchSuffix
+ devel:libsdl2$secondaryArchSuffix
+ devel:libsqlite3$secondaryArchSuffix
+ devel:libssl$secondaryArchSuffix
+ devel:libvorbis$secondaryArchSuffix
+ devel:libz$secondaryArchSuffix
+ "
+
+BUILD_PREREQUIRES="
+ cmd:cmake
+ cmd:gcc$secondaryArchSuffix
+ cmd:ld$secondaryArchSuffix
+ cmd:make
+ cmd:pkg_config$secondaryArchSuffix
+ "
+
+BUILD()
+{
+ mkdir -p build
+ cd build
+
+ cmake .. \
+	-DCMAKE_BUILD_TYPE=STKRelease \
+	-DSTK_INSTALL_DATA_DIR=$dataDir/SuperTuxKart \
+	-DSTK_INSTALL_BINARY_DIR=$appsDir \
+	-DBUILD_RECORDER=OFF \
+	-DBUILD_SHARED_LIBS=OFF
+
+ make $jobArgs
+}
+
+INSTALL()
+{
+ cd build
+ make install
+ addAppDeskbarSymlink "$appsdir/supertuxkart"
```

Comment:
supertuxkart binary needs to be renamed to SuperTuxKart. Also quotes are superfluous here.